src/viz-parole-population/VizParolePopulation.js

```diff
@@ -156,15 +169,26 @@ export default function VizParolePopulation({
   return (
     <VizParolePopulationContainer>
       <VizWrapper>
-        <MapWrapper>
-          <StateDistrictMap
-            data={districtTotals}
-            currentDistrict={districtId}
-            onDistrictClick={onDistrictClick}
-          />
-          <MapCaption>Parole districts in North Dakota</MapCaption>
-        </MapWrapper>
+        <Measure bounds>
+          {({
+            measureRef,
+            contentRect: {
+              bounds: { width },
+            },
+          }) => (
+            <MapWrapper ref={measureRef}>
+              <StateDistrictMap
```

Comment:
The map seems to have stopped filling up the `VizWrapper` on larger screens:

<img width="1018" alt="Screen Shot 2020-07-15 at 7 50 27 PM" src="https://user-images.githubusercontent.com/25503/87610896-a4e41980-c6d4-11ea-9e2f-3307850c6bf7.png">

Reply:
damn you are right ... it flexes down but it never flexes up again 😡